### PR TITLE
Added Parkinson's law

### PR DESCRIPTION
**Pull request by priyankark**
Felt it deserved to be on the list. Contributing as a part of KWOC.

### CODE REVIEW

**Comment by Megha66** [comment]
Relateable!!!

File: readme.md [hunk 1]
```diff
@@ -80,6 +81,14 @@ referred to as `Playing the Hitler card`.
 
 ***
 
+### [Parkinson's law](https://en.wikipedia.org/wiki/Parkinson%27s_law)
+
+> Parkinson's law is the adage that "work expands so as to fill the time available for its completion". It is also sometimes applied to the growth of the bureaucratic apparatus in an organization.
+
+This adage explains why you end up studying the majority of your syllabus one day before the exam despite the fact that you probably had over a week to study.
```
Comment (icyflame) [inline readme.md]: Paragraph should be longer

File: readme.md [hunk 2]
```diff
@@ -80,6 +81,14 @@ referred to as `Playing the Hitler card`.
 
 ***
 
+### [Parkinson's law](https://en.wikipedia.org/wiki/Parkinson%27s_law)
+
+> Parkinson's law is the adage that "work expands so as to fill the time available for its completion". It is also sometimes applied to the growth of the bureaucratic apparatus in an organization.
```
Comment (icyflame) [inline readme.md]: This should be shorter. Preferably only one small sentence that captures the essence of the entry

File: readme.md [hunk 3]
```diff
@@ -80,6 +81,14 @@ referred to as `Playing the Hitler card`.
...
+
+This adage explains why you end up studying the majority of your syllabus one day before the exam despite the fact that you probably had over a week to study.
+
+[R: A Beginners Guide To Parkinson’s Law: How To Do More Stuff By Giving Yourself Less Time](https://impossiblehq.com/parkinsons-law/)
```
Comment (icyflame) [inline readme.md]: Add atleast one more reference

**Review by icyflame — CHANGES_REQUESTED** [review]

**Comment by priyankark** [comment]
Made suggested changes. Please review.

File: readme.md [hunk 4]
```diff
@@ -80,6 +81,16 @@ referred to as `Playing the Hitler card`.
 
 ***
 
+### [Parkinson's law](https://en.wikipedia.org/wiki/Parkinson%27s_law)
+
+> Parkinson's law is the adage that "work expands so as to fill the time available for its completion". 
+
+This adage explains why you end up studying the majority of your syllabus one day before the exam despite the fact that you probably had over a week to study. Interestingly, it is used to explain the inefficiencies of bureaucratic systems. Having earlier deadlines can be a way to get around the problem.
```
Comment (icyflame) [inline readme.md]: wrap at 70 characters

File: readme.md [hunk 5]
```diff
@@ -80,6 +81,16 @@ referred to as `Playing the Hitler card`.
...
+
+[R: A Beginners Guide To Parkinson’s Law: How To Do More Stuff By Giving Yourself Less Time](https://impossiblehq.com/parkinsons-law/)
+
+[R: What is Parkinson's law: The Hindu](http://www.thehindu.com/opinion/op-ed/what-is-parkinsons-law-in-management/article19758089.ece)
```
Comment (icyflame) [inline readme.md]: this reference is unsatisfactory (too short). please find another better reference.

**Review by icyflame — CHANGES_REQUESTED** [review]

**Comment by priyankark** [comment]
Made changes. Please review.

File: readme.md [hunk 6]
```diff
@@ -80,6 +81,21 @@ referred to as `Playing the Hitler card`.
 
 ***
 
+### [Parkinson's law](https://en.wikipedia.org/wiki/Parkinson%27s_law)
```
Comment (icyflame) [inline readme.md]: Four pound signs.

File: readme.md [hunk 7]
```diff
@@ -80,6 +81,21 @@ referred to as `Playing the Hitler card`.
...
+
+> Parkinson's law is the adage that "work expands so as to fill the time available for its completion".
```
Comment (icyflame) [inline readme.md]: No need to put the introductory text, just mention the adage (without double quotes). See other adages for ref

File: readme.md [hunk 8]
```diff
@@ -80,6 +81,21 @@ referred to as `Playing the Hitler card`.
 
 ***
 
+### [Parkinson's law](https://en.wikipedia.org/wiki/Parkinson%27s_law)
+
+> Parkinson's law is the adage that "work expands so as to fill the time available for its completion". 
+
+This adage explains why you end up studying the majority of your syllabus one day before the exam  
+despite the fact that you probably had over a week to study.Interestingly, it is used to
```
Comment (icyflame) [inline readme.md]: Whitespace after periods

**Review by icyflame — CHANGES_REQUESTED** [review]
References are solid. Changes are almost done. Some final problems to be fixed. Thanks for your enthusiasm! :smile:

**Comment by priyankark** [comment]
Thank you for your patience and love the idea for this list!

File: readme.md (hunk 6 above)
Comment (priyankark) [reply readme.md]: Changes made

**Review by icyflame — APPROVED** [review]

**Pull request merged**